.TAOS-CI/.gbs.conf

```diff
@@ -0,0 +1,37 @@
+# TAOS-CI: About a default location of the ".gbs.conf" file
+# If TAOS-CI could not import the (1) ./.TAOS-CI/.gbs.conf or
+# 2) ./packaging/.gbs.conf file, it imports the (3) ~/.gbs.conf file of www-data account.
+# Please keep the ./packaging/.gbs.conf file ASAP for customized GitHub repository.
+
+[general]
+# Current profile name which should match a profile section name
+profile = profile.tizen
+tmpdir = /var/tmp
+editor = vim
+packaging_branch = tizen
+workdir = .
+
+[profile.tizen]
+# Common authentication info for whole profile
+# user = <WRITE_YOUR_TIZEN_ID>
+# passwd = <WRITE_YOUR_TIZEN_PASSSWORD>
+# obs = obs.tizen
+
+# https://source.tizen.org/documentation/developer-guide/environment-setup
```

Comment:
This URL is invalid now. How about changing it like below?
```suggestion
# https://docs.tizen.org/platform/reference/gbs/gbs.conf/
```

Reply:
> https://docs.tizen.org/platform/reference/gbs/gbs.conf/

Thanks. Updated.